editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/adopters/configurators/MavenTemplateConfigurator.java

```diff
@@ -44,13 +45,27 @@ public boolean configure(IProject project, CommonNewProjectMetaData metadata, IP
 		if (ok) {
 			// by default add the maven nature
 			ok = configureMavenNature(project, subMonitor.newChild(1));
+			new BuildAndRefreshJobWaiterUtil().waitJob(subMonitor.newChild(1));
 		}
-		
-		if (ok && !Strings.isBlank(metadata.getCamelVersion())) {
-			// by default configure the version of camel used in the pom.xml
-			ok = MavenUtils.configurePomCamelVersion(project, metadata, metadata.getCamelVersion(), subMonitor.newChild(1));
+
+		// to not trigger too many events while configuring the pom  we reuse the model
+		Model m2model = new CamelMavenUtils().getMavenModel(project);
+
+		try {
+			if (ok) {
+				// by default add staging repos if option enabled
+				ok = MavenUtils.manageStagingRepositories(m2model);
```

Comment:
adding the staging repositories was not done at the same time, maybe something disruptive?

Reply:
I will roll back that class soon.

Reply:
this code is wrong.
MavenUtils.manageStagingRepositories is returning if the m2model has been modified or not.
So in case, staging repo are not enabled it will return false and the creation of the project will skip the other steps and return that the project creation was not done